doc/source/whats_new.rst

```diff
@@ -31,8 +31,9 @@ Changelog
 
     - Add new object :class:`mne.decoding.TimeDecoding` for decoding sensors' evoked response across time by `Jean-Remi King`_
 
-     - Add command ``mne freeview_bem_surfaces`` to quickly check BEM surfaces with Freeview by `Alex Gramfort`_.
+    - Add command ``mne freeview_bem_surfaces`` to quickly check BEM surfaces with Freeview by `Alex Gramfort`_.
 
+    - Add support for splitting epochs into multiple files in :func:`mne.Epochs.save` by `Mainak Jas`_ and `Alexandre Gramfort`_
```

Comment:
@mainakjas did you check to see if this renders properly? We were trying to figure out how to get methods to render, and it would be good to know if `:func:` is it